### PR TITLE
Monika/kava commission withdrawal

### LINKED ISSUE #380: Withdrawal on KAVA only withdraws rewards, no commission
**Describe the bug**
The withdraw button only withdraws the rewards, without the commissions.

**To Reproduce**
Visit the account page

**Expected behavior**
After pressing the 'withdraw' button and signing in the Ledger, all rewards and commissions should be withdrawn.

### PR DESCRIPTION
**Pull request by MonikaCat**
## Description

Fixes #380

## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: imports/ui/accounts/Account.jsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ export default class AccountDetails extends Component {
             totalValueInUSD: 0,
             redelegations: [],
             hasIncentive: false,
+            operator_address : "",
```
Comment (kwunyeung) [inline imports/ui/accounts/Account.jsx:73]: Why this only happens to Kava? I don't see the necessity of putting the `operator_address` here as the commission is retrieved from `getBalance` and passed to `WithdrawButton`.

https://github.com/forbole/big-dipper/blob/d506f71c6003fb6ded2a668d35dc87ff7ee5304f/imports/ui/accounts/Account.jsx#L162
https://github.com/forbole/big-dipper/blob/d506f71c6003fb6ded2a668d35dc87ff7ee5304f/imports/ui/accounts/Account.jsx#L335

Also the `WithdrawButton` only needs to know the `validator_address` as the message only take one value.

The example tx in lcd.kava.forbole.com:1317/txs/57ADDC4CF17861680643A1A2D9DC13E36FD336F204B86B2160C6CB4FD1BF8AFA

```json
      {
          "type": "cosmos-sdk/MsgWithdrawValidatorCommission",
          "value": {
            "validator_address": "kavavaloper14kn0kk33szpwus9nh8n87fjel8djx0y02c7me3"
          }
        }
```

Comment (MonikaCat) [reply imports/ui/accounts/Account.jsx:73]: `Why this only happens to Kava? I don't see the necessity of putting the operator_address here as the commission is retrieved from getBalance and passed to WithdrawButton.`

The commission retrieved from getBalance returns the amount of commission, however, we also need to also pass the validator address hence this value

`Also the WithdrawButton only needs to know the validator_address as the message only take one value.
`
You're right, I will remove those unnecessary values.

Comment (kwunyeung) [reply imports/ui/accounts/Account.jsx:73]: I see. I have mixed up with the variable names. The `operator_address` is actually the `validator_address`.

File: imports/ui/accounts/Account.jsx [hunk 2]
```diff
@@ -683,8 +683,8 @@ export default class AccountDetails extends Component {
                                     {this.state.user ? <Row>
                                         <Col xs={12} >
                                             {this.state.user === this.state.address ? <ClaimSwapButton validator={this.props.validator} address={this.state.operator_address} history={this.props.history} /> : null}
-                                            {this.state.user === this.state.address ? <WithdrawButton history={this.props.history} rewards={this.state.rewards} commission={this.state.commission} address={this.state.operator_address} denom={this.state.denom} /> : null}
-                                            <TransferButton history={this.props.history} address={this.state.address} denom={this.state.denom} total={this.state.total} /></Col>
+                                            {this.state.user === this.state.address ? <WithdrawButton rewards={this.state.rewards} commission={this.state.commission} address={this.state.operator_address} denom={this.state.denom} /> : null}
```
Comment (kwunyeung) [inline imports/ui/accounts/Account.jsx:687]: Why `WithdrawButton` and `TransferButton` don't need the `history` prop but `ClaimSwapButton` needs it? What is the original usage of the `history` prop?

Comment (MonikaCat) [reply imports/ui/accounts/Account.jsx:687]: Oops, I have updated it now. The history prop is used in LedgerButton component, hence I assume the previous developer working on it passed history prop by mistake or needed it in the past, but forgot to clean up the code after.

Comment (kwunyeung) [reply imports/ui/accounts/Account.jsx:687]: Hm... if it's being used in the `LedgerButton` then it should be in all these 3 buttons as they are inherited from `LedgerButton`.

**Pull request merged**